packages/flutter/test/material/autocomplete_test.dart

```diff
@@ -398,4 +401,56 @@ void main() {
     expect(field.controller!.text, 'lemur');
     expect(lastSelection, 'lemur');
   });
+
+  testWidgets('keyboard navigation of the options properly highlights the option', (WidgetTester tester) async {
+
+    void checkOptionHighlight(String label, Color? color) {
+      final RenderBox renderBox = tester.renderObject(find.ancestor(matching: find.byType(Container), of: find.text(label)));
```

Comment:
`tester.renderObject<RenderBox>`

Reply:
Surprised this wasn't caught by the compiler, but thanks.